src/xss-common-qsort.h

```diff
@@ -113,6 +114,19 @@ struct zmm_vector;
 template <typename type>
 struct ymm_vector;
 
+namespace xss{
+namespace avx2{
+template <typename type>
+struct ymm_vector;
```

Comment:
let's rename this to `vex_ymm_vector` to differentiate it from the other one and that way we avoid using namespaces.